### DISCUSSION TITLE
How to make channel priority flexible?

### DISCUSSION THREAD

**Discussion by baichuns** [prefix-dev/pixi#2471] [General]
Hi, I'm getting error like this below,
is excluded because due to strict channel priority not using this option from: 'https://conda.anaconda.org/rapidsai/'

I need to shuffle channels to install packages, is there a way to have channel flexible, so it will look at all channels without priority

thanks

**Comment by ruben-arts** [comment]
Hey @baichuns,

You can disable the priority with:
```toml
[project]
channel-priority = "disabled"
```

reference: https://pixi.sh/latest/reference/project_configuration/#channel-priority-optional
